src/signed.rs

```diff
@@ -37,9 +43,24 @@ impl<T: Signable, S> UncheckedSigned<T, S> {
             key_box,
         })
     }
+
+    pub fn check_partial<'a, MK: MultiKeychain<PartialMultisignature = S>>(
+        self,
+        keychain: &'a MK,
+        indices: &BoolNodeMap,
+    ) -> Result<PartiallyMultisigned<'a, T, MK>, SignatureError<T, S>> {
+        if !keychain.verify_partial(self.signable.hash().as_ref(), &self.signature, indices) {
```

Comment:
Shouldn't it be just verify in fact?

Reply:
Nope. The `UncheckedSigned` is used by both Signed and (Parially)Multisigned